### ISSUE TITLE
Docs for deployment to Google Cloud have errors

### ISSUE THREAD

**Issue by grozdanowski** [strapi/strapi#8114]
**Describe the bug**

The documentation for Google Cloud deployment of Strapi seems to be outdated/wrong for setting the config, so the build fails because config isn't really being used.

**Steps to reproduce the behavior**

1. Go to [docs](https://strapi.io/documentation/3.0.0-beta.x/deployment/google-app-engine.html)
2. Follow steps
3. Lose hours because you cannot understand what you did wrong 😢

**Expected behavior**

Well, instructions should work 😀

**What's the problem**

Basically, in instructions it says to use `config/environments/production/<filename>.json` to override config for specific environment. However, turns out the path is wrong, it should be: `config/env/production/<filename>.json`.
Sounds small but just ate 2.5hrs of my life.

**System**

- Node.js version: 12.18.4
- Strapi version: 3.1.6

**Comment by derrickmehaffy** [comment]
Wrong docs link, you are on the v3 stable, the docs you are referring to are for the beta.

See: https://strapi.io/documentation/v3.x/deployment/google-app-engine.html

**Comment by derrickmehaffy** [comment]
Marking as closed due to wrong docs link, if you still feel there are errors with the new docs please let me know and we can reopen. :smiley:

**Comment by grozdanowski** [comment]
Ugh, my bad. Googled the doc, didn't realise I was looking at the old one.

**Comment by derrickmehaffy** [comment]
No problem we are aware of the SEO issue and are working on it from our side also
